src/rfquack.h

```diff
@@ -140,8 +140,8 @@ uint32_t rfquack_set_status(rfquack_Status *st) {
  * @param addr Address of the register
  * @param value Value of the register
  */
-void rfquack_send_register(rfquack_register_address_t addr,
-                           rfquack_register_value_t value) {
+void rfquack_send_register(uint8_t addr,
```

Comment:
Mhhh, I'm not fond of this. I prefer to keep this generic, so we can have 16 bit register support if we need it. If there isn't a specific reason to remove this, I'd keep it.

Reply:
Hi, I fixed it to uint8_t since RadioLib's SPIreadRegister reads and returns a uint8_t.

Reply:
Let's keep it like this and then cast it to `uint8_t` before calling the various RadioLib methods (i.e., `SPIreadRegister`). For the CC1120 (or other radios with an extended, i.e., > 8 bit, register table) instead of casting, we'll have to make a special `#ifdef` case, but it's not a priority now: just plan for it.